test/integration/testsuite/minishift.go

```diff
@@ -85,6 +85,27 @@ func (m *Minishift) isTheActiveProfile(profileName string) error {
 	return nil
 }
 
+func (m *Minishift) containerStatus(retryCount int, retryWaitPeriod int, imageName string, expected string) error {
+	var containerState string
+	for i := 0; i < retryCount; i++ {
+		runningContainers := m.runner.GetOpenshiftContainers()
+		individualContainer := strings.Split(runningContainers, "\n")
+		for i := 0; i < len(individualContainer); i++ {
```

Comment:
Use this form of `for`  here. https://play.golang.org/p/jpGVFaFQEWH
```
for _, ele := range arr {
```

Also, the variable `i` might override the outer loop `i`.

Reply:
Done